### ISSUE TITLE
Pylon interface dropdowns cut off at bottom

### ISSUE THREAD

**Issue by tbeswick96** [acemod/ACE3#5649]
**Arma 3 Version:** `stable `
**CBA Version:** `stable `
**ACE3 Version:** `3.11.0-rc2`

**Mods:**
```
- CBA_A3
- ace
- CUP
- 3CB (all except units)
- custom mods that add a few pylon magazines
```

**Description:**
- Dropdown boxes for pylon magazines cuts off magazines at the bottom of the list when there are a lot of options.
![](https://goo.gl/EruXDp)

**Comment by 654wak654** [comment]
AFAIK this might be an arma issue, happens in other places too.

What about the position in that screenshot? Does it look that way in eden too?

**Comment by tbeswick96** [comment]
@654wak654 Nope it appears fine in eden...in-fact it seems like another option is visible in eden that isn't show at all in the pylons interface (`1x AGM-114N [ACE]`)
![](https://goo.gl/W3eotE)

**Comment by jonpas** [comment]
Looks like the UI will require bigger changes to fix that properly (it's a vanilla bug really). Pylons can still all be selected with arrow keys for the time being.

**Comment by jonpas** [comment]
Fixed by #5661.
